src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let result = 0;
+  const DAY_COST = 40;
+  let discount = 0;
+
+  for (let n = 1; n <= days; n++) {
+    if (n >= 7) {
+      discount = 50;
+    }
+
+    if (n < 7) {
+      discount = 20;
+    }
 
+    if (n < 3) {
+      discount = 0;
```

Comment:
This condition is also incorrect. The logic for applying discounts should be based on the total number of days, not on each iteration of the loop.